ArduCopter/Copter.h

```diff
@@ -582,6 +582,7 @@ class Copter : public AP_Vehicle {
         uint8_t dynamic_flight          : 1;    // 0   // true if we are moving at a significant speed (used to turn on/off leaky I terms)
         uint8_t in_autorotation         : 1;    // 1   // true when heli is in autorotation
         bool coll_stk_low                  ;    // 2   // true when collective stick is on lower limit
+        bool inverted_flight               ;    // 3   // true when inverted flight option is true
```

Comment:
Rather than adding this state back again I would prefer to add a attitude control method `get_inverted_flight`. That means we only keep the flag in one place, so it can never get out of sync. 

Currently this PR is missing the reset on mode change, asking attitude control would mean that is not a issue:
https://github.com/ArduPilot/ardupilot/blob/d91147552ba03622319bcd31e7651b04d7bfbaa7/ArduCopter/mode.cpp#L463

Reply:
fixed.  Thanks!